akka-docs-dev/rst/java/migration-guide-1.0-2.x-java.rst

```diff
@@ -420,3 +420,21 @@ And use of query parameters from ``Uri`` that looked like this:
 should be replaced by:
 
 .. includecode:: code/docs/MigrationsJava.java#query-param
+
+SynchronousFileSource and SynchronousFileSink
+============================================
+
+Both have been replaced by `Source.file(…)` and `Sink.file(…)`.
+
+Update procedure
+----------------
+
+Replace `SynchronousFileSource.create(` with `Source.file(`
+
+Replace `SynchronousFileSource` with `Source`.
+
+Replace `SynchronousFileSink.create(` with `Sink.file(`
+
+Replace `SynchronousFileSink.appendTo(f)` with `Sink.file(f, true)`
+
+Replace `SynchronousFileSink` with `Sink`
```

Comment:
This won't happen, users only touch the `SynchronousFileSink` object (factory) which returns `Sink`.
The class is internals, see here: https://github.com/akka/akka/pull/18934/files#diff-a5569b40beb4dc7fecd384b9d651c2a1L28 users won't have it in their code.